pkg/controller/storagecluster/update.go

```diff
@@ -627,6 +633,28 @@ func overwriteClusterSpecWithNodeSpec(
 	if nodeSpec.Storage != nil {
 		clusterSpec.Storage = nodeSpec.Storage.DeepCopy()
 	}
+	if nodeSpec.Network != nil {
+		clusterSpec.Network = nodeSpec.Network.DeepCopy()
+	}
+	if len(nodeSpec.Env) > 0 {
+		envMap := make(map[string]*v1.EnvVar)
+		for _, clusterEnv := range clusterSpec.Env {
+			envMap[clusterEnv.Name] = clusterEnv.DeepCopy()
+		}
+		for _, nodeEnv := range nodeSpec.Env {
+			envMap[nodeEnv.Name] = nodeEnv.DeepCopy()
+		}
+		clusterSpec.Env = make([]v1.EnvVar, 0)
+		for _, env := range envMap {
+			clusterSpec.Env = append(clusterSpec.Env, *env)
+		}
+	}
+	if len(nodeSpec.RuntimeOpts) > 0 {
```

Comment:
Can we add in the CRD yaml file, that runtime options in node spec will take precedence.